docs/EXTENDS.md

```diff
@@ -0,0 +1,133 @@
+# Extending Configuration Files
+
+This module supports extending webpack configuration files with
+[ESLint-style](https://eslint.org/docs/user-guide/configuring#extending-configuration-files)
+`extends` functionality. A configuration file can extend the properties and
+collections from base configurations, and are filtered intelligently.
+
+The extends property value is either:
+
+- A `String` that specifies; a fully-qualified configuration path, or an NPM
+module name, or `require` path.
+- An `Array` of `String`. Each additional configuration extends the preceding
+configurations in the array. Configurations are extended recursively so a base
+configuration can also have an extends property.
+- An `Object` containing a `configs` property, which matches the two previously
+types, and a `filters` property, which contains an `Object` specifying filter
+rules.
+
+To demonstrate the concept, suppose we had the following simple setup:
+
+```js
+// base.config.js
+module.exports = {
+  name: 'base',
+  mode: 'development',
+  plugins: [...]
+}
+```
+
+```js
+// webpack.config.js
+module.exports = {
+  extends: path.join(..., 'base-config.js'),
+  name: 'dev'
+}
+```
+
+The resulting configuration object will look like:
+
+```js
+// result
+{
+  name: 'dev',
+  mode: 'development',
+  plugins: [...]
+}
+```
+
+Where `plugins` contains the plugins as defined in the `base.config.js` file.
+
+## Multiple Extended Configurations
+
+There might be situations where we just want to get crazy and extend multiple
+base or nested configurations. In that situation, doing so is as simple as
```

Comment:
Please rephrase the second sentence. _simple_ is too subjective.

Reply:
Can you cite some justification for that, or is it personal preference? Descriptive words like that are meant to normalize a sentence, so that it ready easy rather than terse, and be comforting.

Reply:
Simple is subjective. If user does not understand a piece of code that the writer has declared to be "simple", it is disheartening.

There is lots of content out there telling the same thing. I'm on my mobile right now so I don't have links to paste. I can give you more details on this topic if you'd like to.

Reply:
@valscion while I'm typically loathe to cave to overly-sensitive avoidances such as these, @ooflorent sent me some very good content on alternative phrasing that allows me to keep my special brand of verbiage in the other places mentioned he mentioned 👍 . But in this particular case `'simple'` is accurate, just as ordering socks, dishes, etc is simple.